homeassistant/components/whois/sensor.py

```diff
@@ -189,8 +200,8 @@ async def async_setup_entry(
                 domain=entry.data[CONF_DOMAIN],
             )
             for description in SENSORS
+            if hasattr(data, description.required_attr)
```

Comment:
We shouldn't add these sensors conditionally, instead we should just add them with an unknown status

Reply:
adjusted

Reply:
Just looked at how black formatted it.  ... not so nice